### ISSUE TITLE
Transient mark not working in edit buffer

### ISSUE THREAD

**Issue by pronobis** [alpha22jp/atomic-chrome#17]
When in an atomic-chrome edit buffer, `cua-set-mark` or `activate-mark` do not activate the transient mark (selection/highlighting). However, when `atomic-chrome-enable-auto-update` is set to `nil`, everything works as expected.

**Comment by alpha22jp** [comment]
Hi @pronobis 

Do you mean `cua-set-rectangle-mark` instead of `cua-set-mark`? I'm not that familiar with cua-mode, but `cua-set-mark` looks to be the same as normal `set-mark`.

Anyway, I can select rectangle area with `cua-set-rectangle-mark` and copy it with <kbd>M-w</kbd>, even when `atomic-chrome-enable-auto-update` is `t`.

**Comment by pronobis** [comment]
I meant `cua-set-mark`. Are you using transient mark mode (i.e. do you see your selection highlighted)?

**Comment by alpha22jp** [comment]
Yes, I'm using transient mark mode. I can see the selection highighted when I set mark with `cua-set-mark` and move the cursor.

**Comment by mikeshulman** [comment]
I have this problem too: when `atomic-chrome-enable-auto-update` is `t`, the region is never visibly highlighted in an atomic-chrome buffer.  The issue seems to be that `atomic-chrome-send-buffer-text` deactivates the mark: with  `atomic-chrome-enable-auto-update` set to `nil` if I type `C-x C-s` to manually run `atomic-chrome-send-buffer-text`, the mark also gets deactivated.  I can't figure out exactly why the mark is getting deactivated, though, because it appears to be a heisenbug: if I step through `atomic-chrome-send-buffer-text` with edebug, the mark stays active.
